client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -68,6 +70,8 @@ import './editor.css';
 const MonacoEditor = Loadable(() => import('react-monaco-editor'));
 
 export interface EditorProps {
```

Comment:
You need to import the `LowerJawProps`

```suggestion
export interface EditorProps extends Pick<LowerJawProps, 'challengeMeta' | 'completedPercent'> {
```